tests/integrate_test/cita_charge_mode.sh

```diff
@@ -74,25 +81,29 @@ main() {
     test_fee_back 0
     echo "DONE"
 
-    echo -n "5) Run charge mode tests in v0 ...  "
+    echo -n "5) Check permission denied ... "
```

Comment:
Why does the test be here?

Reply:
And place better?

Reply:
I'll check it later

Reply:
I will refact these tests together, wait some time.

Reply:
ok